### PR TITLE
[IMP] anubia_support module was added

### PR DESCRIPTION
**Pull request by sotogarcia**
Gitlab issue #180: Move module anubia_support to Github

### CODE REVIEW

File: anubia_support/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+│       └───img
+│               anubia_company.png    Anubía company icon
+│               anubia_contact.png    Anubía support user icon
+└───views
+        ir_module_module.xml          Filter to display only the modules from Anubia
+```
+
+## License
+
+**Easy Password Generator** is licensed under the **[GNU Affero General Public License, version 3](http://www.gnu.org/licenses/agpl-3.0.html)** license and all documentation is released under [Creative Commons Attribution-ShareAlike 3.0](http://creativecommons.org/licenses/by-sa/3.0/).
```
Comment (alejandrosantana) [inline anubia_support/README.md]: We are not Easy Password Generator. You can change this to a more generic "This module is licensed ..."
Also, place a LF at the end, please.

File: anubia_support/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+#
+##############################################################################
+
+{
+    'name': "Anubia Support",
+    'version': '0.1',
+
+    'summary': """Anubia, technical support module""",
+
+    'description': """
```
Comment (alejandrosantana) [inline anubia_support/__openerp__.py]: Description here is not needed if there exists a README.rst (.rst, not .md, as Odoo does not support .md). That is what OCA uses right now (take a look at https://github.com/OCA/maintainer-tools/tree/master/template/module ), so enhanced description can be done, and way easier than HTML description (which is also in OCA templates). We could go that route.

File: anubia_support/data/res_partner.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <!--  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+            ADDING ANUBÍA AS PARTNER (Company and Technical Support user)
+
+            - New partner (company) with all the Anubía contact information.
+            - New partner (non company) associated with Technical Support user.
+        - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+
+        <!-- Add Anubía (Company) as partner -->
+        <record model="res.partner" id="anubia_company">
+            <field name="name">Anubia, Soluciones en la Nube, SL</field>
```
Comment (alejandrosantana) [inline anubia_support/data/res_partner.xml]: Typo: "Anubía", not "Anubia" (mind the accented í).

File: anubia_support/data/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            <field name="customer" eval="False"/>
+            <field name="supplier" eval="True" />
+            <field name="is_company" eval="False"/>
+            <field name="street">Calle Colón, 26</field>
+            <field name="street2">Planta 3, Oficina 1</field>
+            <field name="city">Vigo</field>
+            <field name="zip">36201</field>
+            <field name="country_id" ref="base.es" />
+            <field name="phone">+34 986 120 026</field>
+            <field name="mobile">+34 651 089 989</field>
```
Comment (alejandrosantana) [inline anubia_support/data/res_partner.xml]: Nice, but remove the mobile, please.

File: anubia_support/security/res_groups.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <record model="res.groups" id="base.group_user">
+            <field name="users" eval="[(3, ref('anubia_help_desk'))]" />
+        </record>
+
+        <record model="res.groups" id="base.group_system">
+            <field name="users" eval="[(4, ref('anubia_help_desk'))]" />
+        </record>
+
+        <record model="res.groups" id="base.group_no_one">
+            <field name="users" eval="[(4, ref('anubia_help_desk'))]" />
```
Comment (alejandrosantana) [inline anubia_support/security/res_groups.xml]: Shouldn't this be "implied_ids"?

Comment (sotogarcia) [inline anubia_support/security/res_groups.xml]: No!

Comment (alejandrosantana) [inline anubia_support/security/res_groups.xml]: You are right, I have read too fast and misread it.

**Comment by alejandrosantana** [comment]
I will merge and polish the details later on.

**Pull request merged**